ouroboros-network-framework/src/Ouroboros/Network/Channel.hs

```diff
@@ -0,0 +1,313 @@
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
This is not a new file, but a composite of other Channel.hs file in this repo. Is it possible to `git mv` to preserve some history?

Reply:
In this case not I think. Yes, we're copying things from other modules, but not actually entirely deleting the modules where we moved them from.